sct-commons/src/test/java/org/lfenergy/compas/sct/commons/scl/ExtRefServiceTest.java

```diff
@@ -0,0 +1,36 @@
+// SPDX-FileCopyrightText: 2021 RTE FRANCE
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package org.lfenergy.compas.sct.commons.scl;
+
+import org.junit.jupiter.api.Test;
+import org.lfenergy.compas.scl2007b4.model.SCL;
+import org.lfenergy.compas.sct.commons.dto.SclReport;
+import org.lfenergy.compas.sct.commons.testhelpers.SclTestMarshaller;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class ExtRefServiceTest {
+
+    @Test
+    void setExtRefIedNames_should_set_extref() throws Exception {
```

Comment:
I don't really understand this test, are we checking that we could set the extref or check that error occurs and in this case perhaps could we explain why these errors are there.

Reply:
The first part of the user story (issue #77) does not set anything. Setting value happens in part 3 (#79). That 's how the user story is sliced.
This test is for testing errors. 
I agree to add another test for testing the success case where the value is actually set (but it will be a dummy value until we implement #79)

Reply:
I update the code so setExtRefIedNames set the iedName with the FlowID so we can test. This test will be updated in #79